cloudigrade/config/settings/base.py

```diff
@@ -427,14 +394,35 @@ def __print(*args, **kwargs):
 )
 HOUNDIGRADE_SENTRY_ENVIRONMENT = env("HOUNDIGRADE_SENTRY_ENVIRONMENT", default="")
 
+#####################################################################
+# Redis (message queues)
+
+if isClowderEnabled():
+    REDIS_USERNAME = clowder_cfg.inMemoryDb.username
```

Comment:
I also mentioned something about this in the app-interface MR, but just to make sure I cover all the bases…

`inMemoryDb` is somewhat worrying to see since we need our queues to be persistent across restarts. Does this mean something other than what it looks like at first glance?

Reply:
Yea so this is just a super confusing component on bonfire/clowder/something. Its only in memory in name, [the docs touch on there actually being two backing mechanisms](https://redhatinsights.github.io/clowder/clowder/dev/providers/inmemorydb.html), in the case of ephemeral its a pod running redis with a pvc. in stage and prod its our deployed app-sre elasticache clusters. Hopefully this helps clear it up.

Reply:
Thanks for the explanation! 🙂 That seems reasonable. I'm going to ask around elsewhere about those docs because, yeah, that's heckin' confusing. 😕